aten/src/ATen/test/vulkan_test.cpp

```diff
@@ -507,7 +507,7 @@ TEST(VulkanTest, conv2dPrepack) {
       output_min,
       output_max);
   auto tv_out_prepack_ivalues =
-      callOpByName("vulkan::conv2d_clamp_run", "", tv_in, prepack[0]);
+      callOpByName("vulkan_prepack::conv2d_clamp_run", "", tv_in, prepack[0]);
```

Comment:
I think decision regarding these are already made but I am wondering why are the ops no in plain `prepacked` namespace instead of `vulkan_prepack`. I am asking this because we explicitly removed `xnnpack` from the namespace for such prepacking based ops, precisely so that we have the same op that dispatch to different implementation depending on tensor type.

Reply:
Ops have the same name as xnnpack prepacking and for prepack it is called on CPU tensor, thats why I decided to use namespace split with the same naming. Probably conv2d_clamp_run can be in  `prepacked`  namespace as xnnpack,dispatching by DeviceType, but for prepack operation it is on the same DeviceType.